mcp/communications/pilot.c

```diff
@@ -96,9 +97,9 @@ void pilot_compress_and_send(void *telemetries) {
     while (1) {
         // get the current pointer to the pilot linklist
         ll = ll_array[PILOT_TELEMETRY_INDEX];
-        blast_info("serial is 0x%x", *(uint32_t *) ll->serial);
         if (ll != ll_old) {
             if (ll) {
+                blast_info("serial is 0x%x", *(uint32_t *) ll->serial);
```

Comment:
Drop a note here that this is a test print to ensure the serial hashtables match on the GS and the FC